prac_05/emails.py

```diff
@@ -0,0 +1,30 @@
+"""
+Emails
+Estimate: 30 minutes
+Actual:   20 minutes
+"""
+
+
+def main():
+    email = input("Email: ")
+    print(email)
+    name_and_email = {}
+    while email != "":
+        extract_name(email)
+        name_and_email[str(email).strip("[ ]' ' ")] = name
```

Comment:
Maybe a comment explaining the process within this line? Might just make it more clear